vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/addon/spreadsheet/client/SheetTabSheet.java

```diff
@@ -94,10 +94,15 @@ public SheetTabSheet(SheetTabSheetHandler handler) {
 
     private void initDOM() {
         scrollBeginning.setClassName("scroll-tabs-beginning");
+        scrollBeginning.setAttribute("part", "scroll-tab scroll-tab-beginning");
         scrollEnd.setClassName("scroll-tabs-end");
+        scrollEnd.setAttribute("part", "scroll-tab scroll-tab-end");
```

Comment:
Although it's aligned with the class names, it feels like `scroll-tab-start`/`scroll-tab-end` would make more sense, IMO.